### DISCUSSION TITLE
Modification of the tutorial and get error running primal solver

### DISCUSSION THREAD

**Discussion by Ryupuriwatt** [mdolab/dafoam#602] [Tutorials]
I m using a new rotor blade CT-rotor blade using pointwise and then run it just using primal solver. When I check the mesh there was no problem but when I run it I get the following error. It would be very helpful  if profressor can give me any suggest about this error
![image](https://github.com/mdolab/dafoam/assets/115110434/ac3076b3-0a7a-403f-88ef-96467138301f)

**Comment by Ryupuriwatt** [comment]
I do mesh generation by using pointwise, I even try using sanppyhexmesh but also get the same error nan what is the possibly reason so that I can fix this problem

**Comment by friedenhe** [comment]
From the right figure, it seems that your FFD did not fully contain the design surface.

**Reply by Ryupuriwatt** [reply]
Hello now I have fix the problem the problem is from the format of ffd file so I modify ffd generator to my own case and it is working now. I have the second question now I try to run the simulation but I get NAN so I try many mesh to fix it but still doesn't work. Could profressor please give me a possible reason of this problem. I really appriciate your previous answer it help me a lot to point the problem. I think my mesh is not bad but still have no idea how it doesn't works 
![image](https://github.com/mdolab/dafoam/assets/115110434/1a809499-e0c7-4e1a-87c5-b69ca5a765ad)
![image](https://github.com/mdolab/dafoam/assets/115110434/19d2a573-66d8-4aa6-b820-5a4ec39608c7)

**Reply by Ryupuriwatt** [reply]
I am newbie at CFD so 请您多多关照

**Comment by friedenhe** [comment]
What is the total number of mesh cells? You have 400K faces with non-ortho > 70. I think the mesh quality can be improved. Other possible causes of the nan solution are the boundary conditions and fvSolution and fvSchems setup.
